h/templates/client/annotation.html

```diff
@@ -154,18 +154,50 @@
     <div class="annotation-actions" ng-if="!vm.isSaving && !vm.editing() && vm.id()">
       <div ng-show="vm.isSaving">Saving…</div>
       <button class="small btn btn-clean"
-              ng-click="vm.reply()"
-              ><i class="h-icon-reply btn-icon"></i> Reply</button>
+        ng-click="vm.reply()">
+        <i class="h-icon-reply btn-icon"></i>
+        Reply
+      </button>
       <span class="share-dialog-wrapper">
```

Comment:
I think now, or as a follow up PR (you decide) would be a good time to extract this out into its own component. See 'signin-control.{js|html|scss}' for an example of a similar simple component which renders some data and has a couple of events.